geo/src/main/java/org/apache/carbondata/geo/GeoHashImpl.java

```diff
@@ -86,54 +86,54 @@
 
 
   /**
-   * Initialize the geohash index handler instance.
```

Comment:
please rename the class to `GeoHashIndex`

Reply:
Renamed to GeoHashIndex